Installers/Node/index.js

```diff
@@ -99,8 +99,10 @@ function install() {
                     console.log("Injecting index.js");
 
                     var data = fs.readFileSync(_discordPath + _index).toString().split("\n");
-                    data.splice(_importSplice, 0, 'var _betterDiscord = require(\'betterdiscord\');\n');
-                    data.splice(_functionCallSplice, 0, splice);
+                    data = data.join("\n");
+                    data = data.replace('var _ModuleUpdater2 = _interopRequireDefault(_ModuleUpdater);\n','var _ModuleUpdater2 = _interopRequireDefault(_ModuleUpdater);\n\nvar _betterDiscord = require(\'BetterDiscord\');\n var _betterDiscord2;');
```

Comment:
Jiiks dropped all support for the macOS version of BetterDiscord, so I think it's worth while for me to just close this pull request, until such a time that Jiiks cares about the community again.